### PR TITLE
feat: implementation of paid route for event invoices

### LINKED ISSUE #3349: Implement paid route for event invoice
**Is your feature request related to a problem? Please describe.**

Parent Issue - #3284 
_Sub Issue_

As soon as the event invoice payment has been authorized & becomes successful, the user must be redirected to the route where he can view the paid payment for the invoice.

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3349

#### Short description of what this resolves:
As soon as the event invoice payment has been authorized & becomes successful, the user must be redirected to the route where he can view the paid payment for the invoice.

#### Changes proposed in this pull request:
- Adds `/paid` route for event invoice

Screenshot_2019-08-02 Paid Event Invoice -b0929ea6-1b8f-4dea-8075-4f3b437d7325 Open Event

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/event-invoice/invoice-summary.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        {{t 'Your invoice payment completed successfully.'}}
+        <br>
+        {{t 'Find all the details related to your invoice below.'}}
+      </div>
+    </div>
+  </div>
+  <div class="ui secondary segment">
+    <h3 class="weight-400">{{t 'Invoice Details'}}</h3>
+  </div>
+  <table class="ui very basic tablet stackable table order-summary center aligned">
```
Comment (kushthedude) [inline app/templates/components/event-invoice/invoice-summary.hbs]: unstackable

File: app/routes/event-invoice/paid.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    let invoice_identifier = model.get('identifier');
+    return this.l10n.t(`Paid Event Invoice -${invoice_identifier}`);
+  }
+
+  async model(params) {
+    let invoice = await this.store.findRecord('event-invoice', params.invoice_identifier, {
+      include : 'event,user',
+      reload  : true
+    });
+    console.log(invoice.amount);
```
Comment (kushthedude) [inline app/routes/event-invoice/paid.js]: Why this?

File: app/templates/components/event-invoice/event-info.hbs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  </div>
+  <div class="ui padded segment">
+    <strong>{{t 'At'}} {{data.event.locationName}}</strong>
+    <br>
+    <strong>{{t 'From'}}:</strong> {{header-date data.event.startsAt}}
+    <br>
+    <strong>{{t 'To'}}:</strong> {{header-date data.event.endsAt}}
+    {{#if data.event.ownerName}}
+      <br>
+      <strong>{{t 'Organized By'}}:</strong> {{data.event.ownerName}}
```
Comment (kushthedude) [inline app/templates/components/event-invoice/event-info.hbs]: `Owned By`?

File: app/templates/components/event-invoice/invoice-summary.hbs (hunk 1 above)
Comment (shreyanshdwivedi) [reply app/templates/components/event-invoice/invoice-summary.hbs]: @kushthedude if I go with unstackable then I'll have to take care that the table data is not getting out of the table in mobile view.
Moreover, the order page are implemented on stackable concept. I think it'll prevent overflow even if we add few more info to table later

Comment (kushthedude) [reply app/templates/components/event-invoice/invoice-summary.hbs]: @shreyanshdwivedi , There will be loose headers all over which may be much difficult to understand for the user ?

File: app/routes/event-invoice/paid.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+export default class extends Route {
+
+  titleToken(model) {
+    let invoice_identifier = model.get('identifier');
+    return this.l10n.t(`Paid Event Invoice -${invoice_identifier}`);
+  }
+
+  async model(params) {
+    let invoice = await this.store.findRecord('event-invoice', params.invoice_identifier, {
```
Comment (CosmicCoder96) [inline app/routes/event-invoice/paid.js]: why can't you just use 

```js
model(params) {
return this.store.findRecord('event-invoice', params.invoice_identifier, {
include : 'event,user',
      reload  : true
    });
```

File: app/templates/components/event-invoice/invoice-summary.hbs (hunk 1 above)
Comment (shreyanshdwivedi) [reply app/templates/components/event-invoice/invoice-summary.hbs]: Let me add a screenshot for mobile view. I think it is ok
@uds5501 @mrsaicharan1 ?
Screenshot_2019-08-03 Paid Event Invoice -b0929ea6-1b8f-4dea-8075-4f3b437d7325 Open Event

File: app/templates/components/event-invoice/payee-info.hbs [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                      <i class="{{payment-icon data.brand}}"></i>
+                    {{else}}
+                      {{data.brand}}
+                    {{/if}}
+                  </div>
+                </div>
+              </div>
+              <div class="item">
+                <div class="content">
+                  <div class="header">Card Number</div>
```
Comment (CosmicCoder96) [inline app/templates/components/event-invoice/payee-info.hbs]: Translation helpers

File: app/templates/components/event-invoice/payee-info.hbs [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                  <div class="description">**** {{data.last4}}</div>
+                </div>
+              </div>
+              <div class="item">
+                <div class="content">
+                  <div class="header">Exp Date</div>
```
Comment (CosmicCoder96) [inline app/templates/components/event-invoice/payee-info.hbs]: Translation helpers

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 made the changes. Please have a look

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 please review

**Comment by shreyanshdwivedi** [comment]
@kushthedude @uds5501 @mrsaicharan1 please review

**Comment by shreyanshdwivedi** [comment]
@kushthedude @CosmicCoder96  please review

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 please have a look. We cannot merge it even after 3 approvals

**Pull request merged**